<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsClass</class>
 <widget class="QWidget" name="SettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/check.png</normaloff>../icons/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select your language</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="languageComboBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/close.png</normaloff>../icons/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
